### DISCUSSION TITLE
Label Change workflow.

### DISCUSSION THREAD

**Discussion by eha002** [queengooborg/free-dmo-stm32#3] [General]
Whats the best workflow if you constantly use 3 different label types?

is there a way to switch the defaults?

i thought about 3d printing something to supaglue a legit chip to so i can install it with each roll using the Reader intercept.

or having a rotary switch on the back that could switch things over.

I could do super caveman and run multiple Bluepills, but i wonder if it could be done via software.

**Comment by queengooborg** [comment]
I'd say that this is probably something to bring up in the upstream repository -- I know there's a feature request about this in https://github.com/free-dmo/free-dmo-stm32/issues/19.

Given the current setup of this project, my best recommendation is to keep the ST-LINK programmer on standby at all times and re-flash the Bluepill with each SKU.  It's not the best, but it does work!
